ui/raidboss/data/07-dt/raid/r5s.ts

```diff
@@ -243,6 +252,7 @@ const triggerSet: TriggerSet<Data> = {
       outputStrings: {
         insideOut: {
           en: 'Max Melee => Under',
+          cn: '最大近战距离 => 月环',
```

Comment:
```suggestion
          cn: '钢铁 => 月环',
```
Although this is not accurate, it more easy to response quickly.
Melees will know how large is this as they playing savage raid.